flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -63,6 +73,14 @@ class FlxGamepad implements IFlxDestroyable
 	 * Helper class to get the justMoved, justReleased, and float values of analog input.
 	 */
 	public var analog(default, null):FlxGamepadAnalogList;
+	/**
+	 * Helper class to get the float values of motion-sensing input, if it is available
+	 */
+	public var motion(default, null):FlxGamepadMotionValueList;
+	/**
+	 * Helper class to get the float values of mouse-like pointer nput, if it is available
+	 */
+	public var pointer(default, null):FlxGamepadPointerValueList;
```

Comment:
The blame history might not show that because FlxPointer didn't exist back then, but I'm sure you added it specifically for flixel-ui's custom cursor stuff.

That seems a bit misleading then? The usage of "pointer" led me to think that these coords are comparable to the x/y of the mouse / a touch. This at least needs to be documented properly - I don't think the [0, 1] range is mentioned anywhere either?